src/overlays/actors/ovl_En_Nwc/z_en_nwc.c

```diff
@@ -165,8 +165,7 @@ s32 EnNwc_IsFound(EnNwc* this, PlayState* play) {
         return false;
     }
 
-    if (player->stateFlags3 & PLAYER_STATE3_20000000 && // breman mask march
-        this->actor.xzDistToPlayer < 100.0f) {
+    if ((player->stateFlags3 & PLAYER_STATE3_20000000) && this->actor.xzDistToPlayer < 100.0f) {
```

Comment:
Don't think this made it into the header?

Reply:
Do you mean this?
https://github.com/Decompollaborate/mm/blob/1e140522bddf43e10dd573ed28686cd1ef1902ce/include/z64player.h#L584-L585